srcpkgs/intel-media-driver/template

```diff
@@ -13,7 +13,7 @@ maintainer="Stefano Ragni <st3r4g@protonmail.com>"
 license="MIT, BSD-3-Clause"
 homepage="https://github.com/intel/media-driver"
 distfiles="https://github.com/intel/media-driver/archive/intel-media-${version}.tar.gz"
-checksum=1cdd40517d9fee51e3760beea23d2a19c2d5fcb1d6a9ed2bc0af7318d0d3100f
+checksum=491411d6ccdf812e4fe30b870f0623431fe9f14db16fc6b987578769925c8be0
```

Comment:
I got `1c333061479064637ad4ce2a6ea07fd0d4936d4a1da5d462b88a9f5b907ace9a` here :/

Reply:
redownloaded the tarball, I get the same checksum:
```
$ sha256sum hostdir/sources/intel-media-driver-20.3.0/intel-media-20.3.0.tar.gz
491411d6ccdf812e4fe30b870f0623431fe9f14db16fc6b987578769925c8be0  hostdir/sources/intel-media-driver-20.3.0/intel-media-20.3.0.tar.gz
```

Reply:
I will try again in a bit, then.